src/CompositeProducerInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace HelloFresh\Reagieren;
+
+interface CompositeProducerInterface
```

Comment:
Because we have a totally different signature for this interface.... we basically don't need the topic :smile: